packages/web/src/features/landing/frames/NoticeSectionFrame.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import React from "react";
+import styled from "styled-components";
+import { NoticeInfo } from "@sparcs-clubs/web/features/notice/types/notice.type";
+import Link from "next/link";
+import paths from "@sparcs-clubs/web/constants/paths";
+import MoreSectionTitle from "../components/MoreSectionTitle";
+import NoticeCard from "../components/NoticeCard";
+
+const NoticeSectionFrameInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: 70%;
+`;
+
+const NoticeWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-left: 24px;
+`;
+
+type NoticeSectionFrameProps = {
+  noticeList: Array<NoticeInfo>;
+};
+
+const NoticeSectionFrame: React.FC<NoticeSectionFrameProps> = ({
+  noticeList,
+}) => (
+  <NoticeSectionFrameInner>
+    <MoreSectionTitle title="공지사항" />
+    <NoticeWrapper>
+      {noticeList.map(noticeInfo => (
+        <Link
+          href={
+            paths.HOME.path
+          } /* 추후 각 notice에 따른 올바른 path로 수정 바람 --> paths.NOTICE.path + "/" + noticeInfo.id.toString() */
+          style={{ display: "flex", flexDirection: "column" }}
+        >
```

Comment:
이거 안해주면 console 열어보면 `Warning: Each child in a list should have a unique "key" prop.` 에러 날걸요!!
(관련해서 한 번 검색해보세용)

```suggestion
        <Link
          key={noticeInfo.id}
          href={
            paths.HOME.path
          } /* 추후 각 notice에 따른 올바른 path로 수정 바람 --> paths.NOTICE.path + "/" + noticeInfo.id.toString() */
          style={{ display: "flex", flexDirection: "column" }}
        >
```